### PR TITLE
21 create main menu

### PR DESCRIPTION
**Pull request by Anulfo**
## Status
**READY**

## Testing

[ N/A] I have added passing tests for all new functionality.

[ ] I have confirmed that all test prior to this pull request still PASS.

## Documentation

[X ] I have fully documented features in this PR's code. Please put an `X` in the box to confirm.

[ ] This PR does **NOT** require documentation.

## Description
A few sentences describing the overall goals of the pull request's commits.

## Resolves Issue Number
#21 
## Deploy Notes
Notes regarding deployment the contained body of work.  These should note any

```sh
git pull --prune
git checkout 21-Create-Main-Menu
```

## Steps to Test or Reproduce
Outline the steps to test or reproduce the PR here.

1. Get in Visual Studio.
2. Open Solution.
3. Insert this code snippet inside the `Main` Method in Program.cs
```
EmployeeFactory employeeFactory = new EmployeeFactory();

            Employee activeEmployee = employeeFactory.get(2);

            EmployeeFactory.Instance.ActiveEmployee = activeEmployee;

            MainMenu.ReadInput();
```
4. Run and try the different Options. Number 1 and 2 redirects to main menu till new Actions are ready.
Number 3 leads to Administrator View Action.

## Impacted Files in Application
List general components of the application that this PR will affect:

* Actions/MainMenu.cs

### CODE REVIEW

**Review by MattKraatz — APPROVED** [review]
LGTM once we hook up the other actions

**Review by matthamil — APPROVED** [review]
LGTM

**Comment by garrett-vangilder** [comment]
LGTM

**Comment by daynewright** [comment]
LGTM :shipit: !!!!

File: src/customer-service/Actions/MainMenu.cs
Comment (daynewright) [inline src/customer-service/Actions/MainMenu.cs:7]: We have some of these in `customer_service.Actions` and some in `customer_service`.  I think we should decide what we want and make them consistent.  I would suggest we move them all to `Actions`.

@TeamCharles/team  Thoughts?

**Pull request merged**
